micromamba_env.yml

```diff
@@ -0,0 +1,12 @@
+name: protein-runway
+channels:
+  - conda-forge
+  - anaconda
+  - bioconda
+dependencies:
+  - python =3.11
+  - prody
+  - snakemake
+
+  # For MD-TASK:
+  - r-igraph
```

Comment:
This was listed in dependencies, but it might not actually be necessary for our purposes.